esp_flasher/__main__.py

```
@@ -241,11 +241,82 @@ def run_esp_flasher(argv):
     show_logs(stub_chip._port)
 
 
+import os
+import shutil
```

Ruff (0.12.2)

`shutil` imported but unused

Remove unused import: `shutil`

(F401)